src/ChannelServer/Network/Helpers/BuffHelper.cs

```diff
@@ -25,8 +25,8 @@ public static void AddBuff(this Packet packet, Buff buff)
 			packet.PutInt(buff.Duration);
 			packet.PutInt(0);
 			packet.PutInt(0);
-			packet.PutShort((buff.Caster as Character)?.Name.Length + 5 ?? 5);
-			packet.PutString((buff.Caster as Character)?.Name ?? "");
+			packet.PutShort(buff.Caster.Name.Length + 5);
```

Comment:
For future reference, if there's unusual structures like this, where we ourselves aren't even entirely sure why they are the way they are, you should describe what's going on in a short comment. Someone else will look at this code and be really confused what this `+ 5` is about.

Reply:
Sounds good 👍